cmd/build.go

```diff
@@ -0,0 +1,64 @@
+package cmd
+
+import (
+	"fmt"
+	"io/ioutil"
+	"os"
+	"path/filepath"
+
+	"github.com/spf13/cobra"
+	"github.com/turnerlabs/fargate-create/cmd/build"
+)
+
+var buildCmd = &cobra.Command{
+	Use:   "build",
+	Short: "Scaffold out artifacts for various build systems",
+	Args:  cobra.ExactArgs(1),
+	Run:   doBuild,
+	Example: `
+fargate-create build local
+fargate-create build circleciv2
+`,
+}
+
+func init() {
+	rootCmd.AddCommand(buildCmd)
+}
+
+func doBuild(cmd *cobra.Command, args []string) {
+
+	//load build provider
+	providerString := args[0]
+	provider, err := build.GetProvider(providerString)
+	check(err)
```

Comment:
For a minute I thought you were using a Go 2 compiler prototype with `check` for error handling, but no, just a private function.  Nothing to see here; carry on. :)